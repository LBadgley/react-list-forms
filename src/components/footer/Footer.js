import React from 'react';
import styles from './Footer.css';

export default function Footer() {
  return (
    <footer className={styles.Footer}>
      <p>by Laura Badgley</p>
    </footer>
  );
}
