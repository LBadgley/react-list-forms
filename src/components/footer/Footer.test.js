import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('footer test', () => {
  it('renders the footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
