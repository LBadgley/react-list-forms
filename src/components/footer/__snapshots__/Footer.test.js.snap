// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer test renders the footer 1`] = `
<footer
  className="Footer"
>
  <p>
    by Laura Badgley
  </p>
</footer>
`;
